lib/bolt/plugin.rb

```diff
@@ -188,6 +198,13 @@ def add_module_plugin(plugin_name)
         config: config_for_plugin(plugin_name)
       }
 
+      # Make sure that the plugin's config is validated _before_ these checks.
```

Comment:
I don't understand the context of this comment. It seems like it's referring to the fact that we call `config_for_plugin` above. Is that the correct interpretation? If so, I think it should be moved to that line and rephrased because it seems ambiguous here.

Reply:
Yeah, I can rephrase it. It's mostly for people (like me) who may wonder why we're validating config for a potentially unknown plugin.